code/modules/surgery/advanced/brainwashing.dm

```diff
@@ -20,8 +20,8 @@
 /datum/surgery/advanced/brainwashing/can_start(mob/user, mob/living/carbon/target)
 	if(!..())
 		return FALSE
-	var/obj/item/organ/brain/B = target.getorganslot(ORGAN_SLOT_BRAIN)
-	if(!B)
+	var/obj/item/organ/brain/brain = target.getorganslot(ORGAN_SLOT_BRAIN)
```

Comment:
these should be more descriptive https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
basically all of them are like this

Reply:
target_brain?

Reply:
for example yes
sadly they'll have to change all the other vars to be more descriptive and not all can be just target_brain